describe/AbstractDataAttribute.ts

```diff
@@ -0,0 +1,112 @@
+import { DAType, DATypeDescription } from "./DAType.js";
+import { EnumType, EnumTypeDescription } from "./EnumType.js";
+import { NamingDescription, describeNaming } from "./Naming.js";
+import { ValDescription, describeVal } from "./Val.js";
+
+export function isAbstractDataAttributeDescription(
+  type: AbstractDataAttributeDescription
+): type is AbstractDataAttributeDescription {
```

Comment:
Is this type guard actually of use to you?

I would have assumed that in order for this to be of any use it would require the signature
```suggestion
export function isAbstractDataAttributeDescription(
  type: Description
): type is AbstractDataAttributeDescription {
```